### PR TITLE
Fix import of collections

### PR DESCRIPTION
**Pull request by gonuke**
Beginning with python 3.8, the `collections` module needs to be imported differently.  This was identified in #1232 .

This should replace all imports to `collections` to be safe for the changes in python3.8.

It also removes some places where `collections` was imported and never used.

### CODE REVIEW

File: pyne/openmc_utils.py [hunk 1]
```diff
@@ -5,7 +5,10 @@
 import io
 import sys
 from warnings import warn
-from collections import namedtuple
+try:
+    from collections.abc import namedtuple
+except ImportError:
+    from collections import namedtuple
```
Comment (bam241) [inline pyne/openmc_utils.py:11]: I think this one did't need to be changed. as it does not rely on `abc` in the first place.

File: pyne/rxdata.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import collections
```
Comment (bam241) [inline pyne/rxdata.py:1]: needed L19

**Review by bam241 — APPROVED** [review]
LGTM!
thx @gonuke

**Comment by bam241** [comment]
@gonuke could you verify you agree with my conflict resolution https://github.com/pyne/pyne/pull/1233/commits/d7ea237bfc8bf6c8cebe3d369979109358c4a12b

(basically removed the outdated `MaterialLibrary`  implementation in `material.pyx`)

if you agree, I am ready to merge.

**Comment by gonuke** [comment]
I had to add the fix to your new `material_library` code

**Comment by bam241** [comment]
LGTM!,
thx @gonuke !
merging

**Pull request merged**